package.json

```
@@ -80,6 +80,7 @@
   },
   "jest": {
     "testPathIgnorePatterns": [
+      "frontend",
```

```suggestion
      "/frontend/",
```

if we're placing before `/node_modules/`.